var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -369,6 +369,11 @@ class Llvm(CMakePackage, CudaPackage):
     # make libflags a list in openmp subproject when ~omp_as_runtime
     patch('libomp-libflags-as-list.patch', when='@3.7:')
 
+    # Add missing include leading to build fail with clang
+    patch('https://github.com/llvm/llvm-project/commit/b498303066a63a203d24f739b2d2e0e56dca70d1.patch',
+          sha256='151d0fb4c32933d224033c2fd855b28fe85c7d487307c6345c7074c19d77feb5',
+          when='@8.0.0:11.1.0')
```

Comment:
Since this covers the full `8.x` - `11.x` range, the `when` clause can be slightly simplified with `when='@8:11'`.  Even though the end result is the same, using `11.1.0` implies the existence of later `11.x` release where this is already fixed.